libbarrett_config/right_arm/calibration.conf

```diff
@@ -30,7 +30,8 @@ zerocal:
 	
 	wam7w:
 	(
-		(1, ( 3.14, -1.57,   0.0,  1.57,   0.0,   0.0,   0.0 ), "the right J1 scribe marks are aligned"),
+		# (1, ( 3.14, -1.57,   0.0,  1.57,   0.0,   0.0,   0.0 ), "the right J1 scribe marks are aligned"),
```

Comment:
Double-check which one's correct and remove this line.

Reply:
removed the lines.